docker/buoy/Dockerfile

```diff
@@ -30,4 +42,4 @@ RUN /bin/bash -c 'source /opt/ros/${ROS_DISTRO}/setup.bash \
   && cd /tmp/buoy_ws \
   && colcon build'
 
-ENTRYPOINT ["/bin/bash" , "-c" , "source /tmp/buoy_ws/install/setup.bash && /bin/bash"]
+ENTRYPOINT ["/bin/bash" , "-c" , "source /tmp/buoy_ws/install/setup.bash &&  && /bin/bash"]
```

Comment:
Is this extra `&&` a typo or did you want to put another command here? I could just push and approve but... in case you actually wanted to add something.

Reply:
sorry, typo